tests/Collections/PropertyCollectionTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+require('vendor/autoload.php');
+
+use ArmadilloEstates\Collections\PropertyCollection;
+
+class PropertyCollectionTest extends PHPUnit\Framework\TestCase
+{
+    public function testSuccessPropertyCollection()
+    {
+        $entity = $this->createMock(\ArmadilloEstates\Entities\PropertyEntity::class);
+        $array = [$entity, $entity, $entity];
+        $collection = new PropertyCollection($array);
+        $this->addToAssertionCount(1);
```

Comment:
you dont need to use this. look at:
https://phpunit.readthedocs.io/en/9.0/assertions.html#assertinstanceof